5.5/run-mysqld.sh

```diff
@@ -78,21 +78,26 @@ if [ ! -d '/var/lib/mysql/mysql' ]; then
 	admin_flags="--defaults-file=/opt/openshift/etc/my.cnf $mysql_flags"
 
 	mysqladmin $admin_flags -f drop test
-	mysqladmin $admin_flags create "${mysql_db}"
+	mysqladmin $admin_flags create "${MYSQL_DATABASE}"
 	mysql $mysql_flags <<-EOSQL
-		CREATE USER '${mysql_user}'@'%' IDENTIFIED BY '${mysql_pass}';
-		GRANT ALL ON \`${mysql_db}\`.* TO '${mysql_user}'@'%' ;
+		CREATE USER '${MYSQL_USER}'@'%' IDENTIFIED BY '${MYSQL_PASSWORD}';
+		GRANT ALL ON \`${MYSQL_DATABASE}\`.* TO '${MYSQL_USER}'@'%' ;
 		FLUSH PRIVILEGES ;
 	EOSQL
 
-	if [ -v root_pass ]; then
+	if [ -v MYSQL_ROOT_PASSWORD ]; then
 		mysql $mysql_flags <<-EOSQL
-			GRANT ALL PRIVILEGES ON *.* TO 'root'@'%' IDENTIFIED BY '${root_pass}';
+			GRANT ALL PRIVILEGES ON *.* TO 'root'@'%' IDENTIFIED BY '${MYSQL_ROOT_PASSWORD}';
 		EOSQL
 	fi
 	mysqladmin $admin_flags flush-privileges shutdown
+
+	unset_env_vars
+
+	exec /opt/rh/mysql55/root/usr/libexec/mysqld \
+		--defaults-file=/opt/openshift/etc/my.cnf \
```

Comment:
I'll just add a defaults file env var, nothing else we use /opt/openshift for anyway, and it might go away soon in any case.